tests/conftest.py

```diff
@@ -1586,7 +1586,7 @@ def fixture_aws_credentials() -> Generator[None, None, None]:
     os.environ.pop("AWS_DEFAULT_REGION")
 
 
-MOTO_SERVER = ThreadedMotoServer(port=5000)
+MOTO_SERVER = ThreadedMotoServer(ip_address="localhost", port=5000)
```

Comment:
The default ip address of the moto server is `0.0.0.0`. Since we only use it for unit test, I think it make more sense to start it at `localhost`